### PR TITLE
Usage of Poetry

### PR DESCRIPTION
**Pull request by benoit9126**
- I drop the `setup.py` file and the `Manifest.in` files which are not anymore necessary
- I mark the protobuf Python generated file as excluded from the coverage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +19,16 @@ Note that `mapbox-vector-tile` depends on [Shapely](https://pypi.python.org/pypi
...
+following keys
 
 * `name`: layer name
 * `features`: an array of features. A feature is a dictionary with the following keys:
 
   * `geometry`: representation of the feature geometry in WKT, WKB, or a shapely geometry. Coordinates are relative to the tile, scaled in the range `[0, 4096)`. See below for example code to perform the necessary transformation. *Note* that `GeometryCollection` types are not supported, and will trigger a `ValueError`.
   * `properties`: a dictionary with a few keys and their corresponding values.
 
-```python
+```pycon
```
Comment (rmarianski) [inline README.md:31]: what does this do differently?

File: README.md [hunk 2]
```diff
@@ -188,13 +190,14 @@ mapbox_vector_tile.encode([
 
 In this example, the coordinate that would get encoded would be (2048, 2048)
 
-Additionally, if the data is already in a cooridnate system with y values going down, the encoder supports an option, `y_coord_down`, that can be set to True. This will suppress flipping the y coordinate values during encoding.
+Additionally, if the data is already in a coordidnate system with y values going down, the encoder supports an
```
Comment (rmarianski) [inline README.md:193]:
```suggestion
Additionally, if the data is already in a coordinate system with y values going down, the encoder supports an
```

**Review by rmarianski — COMMENTED** [review]
does this retain backwards compatibility with the setuptools ecosystem?

File: README.md (hunk 1 above)
Comment (benoit9126) [reply README.md:31]: Please excuse me. It was for my IDE when I did automatic replacement (eg. `addFeatures` ⇾ `add_features`). I will revert these changes in the README file.

**Review by rmarianski — APPROVED** [review]

**Pull request merged**